1.0/docs/devguide/templates.md

```diff
@@ -303,6 +303,30 @@ If you or a third-party library mutate the array without Polymer's methods,
 you need to call [`notifySplices`](#notifysplices) to ensure that any elements
 watching the array are properly notified.
 
+### Performance properties {#dom-repeat-perf}
```

Comment:
I think the task here would be "Improve performance for large lists" or something of that nature.

Main point: By default, `dom-repeat` tries to render all of the list items at once. If you try to use `dom-repeat` to render a very large list of items, the UI may freeze while it's rendering the list. If you encounter this problem, enable "chunked" rendering by setting `initialCount`. In chunked mode, `dom-repeat` renders `initialCount` items at first, then renders the rest of the items incrementally one chunk per animation frame. This lets the UI thread to handle user input between chunks. 

`dom-repeat` adjusts the number of items rendered in each chunk to try and maintain a target framerate.
You can further tune rendering by setting `targetFramerate` and `delay`.